frontend/libs/studio-components/src/components/StudioIconTextfield/StudioIconTextfield.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { StudioTextfield, type StudioTextfieldProps } from '../StudioTextfield';
+
+import classes from './StudioIconTextfield.module.css';
+
+export type StudioIconTextfieldProps = {
+  icon: React.ReactNode;
+} & StudioTextfieldProps;
+
+export const StudioIconTextfield = ({
+  icon,
+  className,
+  ...rest
+}: StudioIconTextfieldProps): React.ReactElement => {
+  return (
+    <div className={classes.container}>
+      <div aria-hidden className={classes.prefixIcon}>
+        {icon}
+      </div>
+      <StudioTextfield {...rest} className={`${className} ${classes.textfield}`} />
```

Comment:
We can compose className by using a variable.

```typescript
const className = cn([givenClassName, classes.textfield])
```

```suggestion
      <StudioTextfield {...rest} className={className} />
```